### ISSUE TITLE
Installation error for wav2letter v0.2

### ISSUE THREAD

**Issue by amanjain-1997** [facebookresearch/wav2letter#952]
### Question
Getting this error while installing wav2letter v0.2 already installed flashlight 0.2

[  4%] Building CXX object CMakeFiles/wav2letter++.dir/src/common/FlashlightUtils.cpp.o
In file included from /usr/local/include/flashlight/memory/memory.h:11:0,
                 from /usr/local/include/flashlight/flashlight.h:15,
                 from /home/aman/Desktop/wav2letter/wav2letter/src/common/FlashlightUtils.h:20,
                 from /home/aman/Desktop/wav2letter/wav2letter/src/common/FlashlightUtils.cpp:9:
/usr/local/include/flashlight/memory/MemoryManagerAdapter.h:11:10: fatal error: af/memory.h: No such file or directory
 #include <af/memory.h>
          ^~~~~~~~~~~~~
compilation terminated.
CMakeFiles/wav2letter++.dir/build.make:458: recipe for target 'CMakeFiles/wav2letter++.dir/src/common/FlashlightUtils.cpp.o' failed
make[2]: *** [CMakeFiles/wav2letter++.dir/src/common/FlashlightUtils.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
CMakeFiles/Makefile2:520: recipe for target 'CMakeFiles/wav2letter++.dir/all' failed
make[1]: *** [CMakeFiles/wav2letter++.dir/all] Error 2

#### Additional Context
[Add any additional information here]

**Comment by avidov** [comment]
MemoryManagerAdapter exists in a newer Flashlight release.
In that new release wav2letter is incorperated into flashlight under flashlight/app/asr directory
Please try to either to pull an older version of Flashlight, one that works with wav2letter.
Or use the latest Flashlight release and use the wav2letter now under flashlight/app/asr .

**Comment by amanjain-1997** [comment]
what flashlight version should i use for wav2letter v0.2.
The flashlight 0.2 has MemoryManagerAdapter
https://github.com/facebookresearch/flashlight/tree/v0.2/flashlight/memory

> MemoryManagerAdapter exists in a newer Flashlight release.
> In that new release wav2letter is incorperated into flashlight under flashlight/app/asr directory
> Please try to either to pull an older version of Flashlight, one that works with wav2letter.
> Or use the latest Flashlight release and use the wav2letter now under flashlight/app/asr .

**Comment by tlikhomanenko** [comment]
Yep, you are using correct, in both repos, w2l and fl, branch v0.2.

Could you check that you linked arrayfire both for fl and w2l cmake? Also can you confirm that all fl test pass for you?
